src/appleseed/renderer/kernel/lighting/lighttree.cpp

```diff
@@ -354,45 +354,45 @@ float LightTree::compute_node_probability(
     const ShadingPoint&             shading_point) const
 {
     // Calculate probability of a single node based on its contribution over solid angle.
-    const float r = bbox.radius();
-    const float r2 = r * r;
+    const float r2 = bbox.square_radius();
     const float rcp_surface_area = 1.0f / r2;
 
-    // Triangle centroid is a more precise pocision than the center of the bbox.
+    // Triangle centroid is a more precise position than the center of the bbox.
     const Item& item = m_items[node.get_item_index()];
-    const Vector3d position = (node.is_leaf() && item.m_light_type == EmittingTriangleType)
+    const Vector3d& position = (node.is_leaf() && item.m_light_type == EmittingTriangleType)
```

Comment:
That's not correct since both `emitting_triangle_centroid()` and `bbox.center()` return a new vector.